Sources/VLCOneDriveTableViewController.m

```diff
@@ -127,7 +127,7 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         self.title = selectedItem.name;
     } else {
         VLCMediaList *mediaList;
-        NSURL *url = [NSURL URLWithString:selectedItem.dictionaryFromItem[@"@content.downloadUrl"]];
+        NSURL *url = [NSURL URLWithString:selectedItem.webUrl];
```

Comment:
Have you tested this?

I remember trying this before the sad realisation that I needed to use the dictionary. 😢

Reply:
yes tested it

Reply:
This doesn't work for me.

Reply:
streaming